src/patterns/ordo-mapping-codes_relationship-curies.csv

```diff
@@ -0,0 +1,6 @@
+﻿ordo_mapping_code,relationship_curie
+E,skos:exactMatch
+NTBT,skos:narrowMatch
+BTNT,skos:broadMatch
+ND,skos:related
+W,owl:disjointWith
```

Comment:
Sure thing. I'll add this as one of the sub-steps in [the github issue](https://github.com/monarch-initiative/mondo-ingest/issues/23) so that we don't forget to address.